README.md

```diff
@@ -68,6 +68,43 @@ const { success, error } = await storage.delete(key, { shared });
 
 </details>
 
+
+## Operations counter
+
+<details>
+<summary>A tool that provide the ability to count the number of operations base on a pre-defined period.</summary>
+
+- This is the way to count operations around your project.
+- Has 3 pre-defined periods
+  - `DAILY` - the counter will be reset every day (base on GMT time zone)
+  - `MONTHLY` - the counter will be reset every month (base on GMT time zone).
+  - `YEARLY` -  the counter will be reset every year (base on GMT time zone).
```

Comment:
What does it mean `base on GMT time zone`?

Reply:
@gregra81 
Greenwich Mean Time (GMT) 
universal time
GMT 0
https://www.google.com/search?q=gmt+time+now&oq=hgmt&gs_lcrp=EgZjaHJvbWUqDwgEEAAYChiLAxixAxiABDIGCAAQRRg5MhIIARAAGAoYgwEYsQMYyQMYgAQyDwgCEAAYChiDARixAxiABDIMCAMQABgKGIsDGIAEMg8IBBAAGAoYiwMYsQMYgAQyEggFEAAYChiDARiLAxixAxiABDIJCAYQABgKGIAEMgkIBxAAGAoYgAQyCQgIEAAYChiABDIMCAkQABgKGJIDGIAE0gEINTIzNGoxajGoAgCwAgA&sourceid=chrome&ie=UTF-8

https://www.google.com/search?q=universal+time+now+VS+GMT&sca_esv=589592780&ei=JfR1ZfeQHp-lkdUPzcOcqAs&ved=0ahUKEwi3t4SlsYWDAxWfUqQEHc0hB7UQ4dUDCBA&uact=5&oq=universal+time+now+VS+GMT&gs_lp=Egxnd3Mtd2l6LXNlcnAiGXVuaXZlcnNhbCB0aW1lIG5vdyBWUyBHTVQyBRAhGKABMgUQIRigATIFECEYoAEyBRAhGKABMggQIRgWGB4YHUjpQ1CwFlibL3AHeAGQAQGYAc4DoAGAEaoBCTAuNC4yLjIuMbgBA8gBAPgBAcICChAAGEcY1gQYsAPCAg0QABiABBiKBRhDGLADwgIFEAAYgATCAgYQABgWGB7CAgoQIRgWGB4YDxgdwgIEECEYFeIDBBgAIEGIBgGQBgo&sclient=gws-wiz-serp

Reply:
```
  - `DAILY` - the counter will be reset every day (at 00:00 GMT)
  - `MONTHLY` - the counter will be reset every month
  - `YEARLY` -  the counter will be reset every year
```